setup.py

```diff
@@ -1,139 +1,2 @@
-#############################################################################
-# Copyright (c) 2018, Voilà Contributors                                    #
-# Copyright (c) 2018, QuantStack                                            #
-#                                                                           #
-# Distributed under the terms of the BSD 3-Clause License.                  #
-#                                                                           #
-# The full license is in the file LICENSE, distributed with this software.  #
-#############################################################################
-from pathlib import Path
-from urllib.request import urlopen
-import setuptools
-import sys
-import os
-
-HERE = Path(__file__).parent.resolve()
-
-# The name of the project
-NAME = "voila"
-
-labext_name = "@voila-dashboards/jupyterlab-preview"
-lab_extension_dest = HERE / NAME / "labextension"
-
-# Representative files that should exist after a successful build
-ensured_targets = [
-    str(HERE / "share/jupyter/voila/templates/base/static/voila.js"),
-    str(lab_extension_dest / "package.json"),
-]
-
-
-data_files_spec = [
-    (
-        "etc/jupyter/jupyter_server_config.d",
-        "etc/jupyter/jupyter_server_config.d",
-        "voila.json",
-    ),
-    (
-        "etc/jupyter/jupyter_notebook_config.d",
-        "etc/jupyter/jupyter_notebook_config.d",
-        "voila.json",
-    ),
-    (
-        "etc/jupyter/nbconfig/notebook.d",
-        "etc/jupyter/nbconfig/notebook.d",
-        "voila.json",
-    ),
-    ("share/jupyter/nbextensions/voila", "voila/static", "extension.js"),
-    ("share/jupyter/labextensions/%s" % labext_name, str(lab_extension_dest), "**"),
-    ("share/jupyter/labextensions/%s" % labext_name, str(HERE), "install.json"),
-    ("share/jupyter/voila/templates", "share/jupyter/voila/templates", "**/*[!.map]"),
-]
-
-
-jupyterlab_apputils_version = "3.2.8"
-jupyterlab_theme_light_version = "3.2.8"
-
-css_files = [
-    (
-        "https://unpkg.com/@jupyterlab/apputils@%s/style/materialcolors.css"
-        % jupyterlab_apputils_version,
-        "materialcolors.css",
-    ),
-    (
-        "https://unpkg.com/@jupyterlab/theme-light-extension@%s/style/variables.css"
-        % jupyterlab_theme_light_version,
-        "labvariables.css",
-    ),
-]
```

Comment:
It looks like some data files and some CSS is not installed anymore? At least that's what I see from the code.

Reply:
Data files should still be handled by the new `tool.hatch.build.targets.wheel.shared-data` config.

Good catch for the CSS files, need to check how to keep them.

Reply:
Looks like we can indeed keep the same logic with a custom hook in `hatch_build.py`.